Mage.Tests/src/test/java/org/mage/test/cards/cost/adventure/AdventureCardsTest.java

```diff
@@ -397,7 +397,7 @@ public void testRimrockKnightPermanentText() {
         assertGraveyardCount(playerA, 0);
 
         Permanent rimrock = getPermanent("Rimrock Knight");
-        Assert.assertTrue(rimrock.getRules(currentGame).get(0).startsWith("Adventure Instant")); // must have adventure spell info on battlefield
+        Assert.assertTrue(rimrock.getRules(currentGame).get(0).startsWith("Target creature")); // must have adventure spell info on battlefield
```

Comment:
Is it show full text (adventure and main spell) in card hint popup and in text only mode (if you delete image and look at card hint panel on the right side of the game panel/chat or on the left side of the deck editor)?